workflows/crisprseq_screening.nf

```diff
@@ -77,22 +76,28 @@ workflow CRISPRSEQ_SCREENING {
 
     if(!params.count_table){
         //
-        // SUBWORKFLOW: Read in samplesheet, validate and stage input files
+        // Create input channel from input file provided through params.input
         //
-        INPUT_CHECK_SCREENING (
-            file(params.input)
-        )
-        ch_versions = ch_versions.mix(INPUT_CHECK_SCREENING.out.versions)
+        Channel.fromSamplesheet("input")
+        .map{ meta, fastq_1, fastq_2, x, y, z ->
```

Comment:
We don't use it currently, the only difference between single and paired end is a parameter in a module further in the analysis, but both types are treated equally otherwise.

Reply:
Thanks to this comment I've noticed I added the `single_end` label wrong for the other channel, and now I have a bug, so not ready yet 😓

Reply:
Ai, good luck with bug fixing!